### PR TITLE
Added devfile for Stonesoup Onboarding

### PR DESCRIPTION
**Pull request by NikishaPatil**

### CODE REVIEW

**Comment by NikishaPatil** [comment]
@jsenko @riprasad Please review and merge.

File: devfile.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+schemaVersion: 2.2.0
+metadata:
+  name: fleet-manager
+  version: 0.1.0
+  provider: Red Hat
+  icon: https://design.jboss.org/quarkus/logo/final/SVG/quarkus_icon_rgb_default.svg
```
Comment (jsenko) [inline devfile.yaml:6]: We have our logos btw https://design.jboss.org/apicurio/

File: devfile.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  language: Java
+  attributes:
+    alpha.dockerimage-port: 8081
+parent:
+  id: java-quarkus
+  registryUrl: 'https://registry.devfile.io'
+components:
+  - name: image-build
+    image:
+      imageName: java-quarkus-image:latest
```
Comment (jsenko) [inline devfile.yaml:22]: FM is currently based on Quarkus `2.7.5.Final`, will using the `latest` image cause any issues?

Comment (NikishaPatil) [reply devfile.yaml]: My bad, this represents the name of the image that we build. Updated with the correct image name

**Review by jsenko — APPROVED** [review]
LGTM

File: devfile.yaml (hunk 1 above)
Comment (NikishaPatil) [reply devfile.yaml]: Thanks. Updated the url to use Apicurio Logo

**Comment by jsenko** [comment]
@riprasad Can you please double-check?

**Comment by riprasad** [comment]
LGTM. Thank you @NikishaPatil 
Merging.

**Pull request merged**